contracts/SplitMerge.sol

```diff
@@ -138,6 +138,14 @@ contract SplitMerge is Initializable, Ownable {
     return packageToGeohashes[_packageToken];
   }
 
+  function getPackageCount(uint256 _packageToken) public view returns (uint256) {
+    return packageGeohashesCount[_packageToken];
+  }
```

Comment:
You should be able to access geohashesCount for a packageToken both from web3 and another contract since it's public.

Reply:
Just use a field name as a method and a tokenId as an argument with web3 like:
```javascript
await contractInstance.methods.packageGeohashesCount(tokenId).send();
```

Reply:
@chebykin i was try to access geohashesCount mapping from another contract, but received error. I think no way to do this